src/ResultWriter/FaultWriterC.cpp

```diff
@@ -36,6 +36,8 @@
  *
  * @section DESCRIPTION
  */
+#if 0
```

Comment:
just remove then?

Reply:
It will be removed later. I gave my reasoning below